### ISSUE TITLE
[1.10.2] Digital Miner crashing the game

### ISSUE THREAD

**Issue by Thetos7** [aidancbrady/Mekanism#4787]
*Please use the search functionality before reporting an issue. Also take a look at the closed issues!*

#### Issue description:
When using the digital miner, right after hitting start the game crashes.
I might have to not that my game has quite a few mods, but I don't feel it's coming for the other mods :p

#### Steps to reproduce:
1.setup a digital miner normally
2.hit start
3.game crash

#### Version (make sure you are on the latest version before reporting):
**Forge:12.18.3.2511** 
**Mekanism:1.10.2-9.2.4.103** 
**Other relevant version:** 

#### If a (crash)log is relevant for this issue, link it here: (_It's almost always relevant_)
https://gist.github.com/Thetos7/8e412221c4d338b0d19bce6d23402e4b
(I have put the entire log, but I think you might only need the error report near the end of it :p)
[gist/pastebin/etc link here]

**Comment by MalcolmAnderson** [comment]
I'm curious as to how to debug this.
In the crash log there's a ton of interesting stuff
@Line 2523 there start to be a small ton of texture issues
@Line 2593 The system seems to recover the textures from enderio

@Line 2782 we have a "Ticking block entity" caused by a NullPointerException (@Line 2788)

But the bottom line diagnosis in the crash log starts at @2838
And it does implicate the digital miner (@2853)

What should I be looking at?

Lastly, @Thetos7 have you been able to reproduce the issue with the current build of Mekanism?

**Comment by Thetos7** [comment]
After some more testing, I think it's not really/ entirely the digital miner that is at fault for the crash.
from what I have tested, the crash occurs when the digital miner tries to pull a block for a storage drawer with the auto-pull.
(I have verified by putting a vanilla chest instead of the drawer and it worked without any crash)
For these tests, I have been using the latest versions available on curse forge (ver. 9.2.4.103 for 1.10.2).
I think the NullPointerException mentioned @line 2788 may be causing the crash (if it is at all mekanism related).
I think I must apologize for this since it may not be a problem with mekanism. (I'm really sorry ^^')
